### PR TITLE
feat: added java instrumentation for jdbc execute batch

### PR DESCRIPTION
**Pull request by hsinghkalsi**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ X] Did you check you NRQL syntax? - Does it work?
- [ X] Did you check your dashboard image quality? -  Do they look good?
- [ x] Did you check that your alerts actually work?
- [ x] Did you include an InstallPlan and Documentation reference?
- [ x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/java/jdbc-executebatch/config.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    Once deployed the instrumentation will start to track the call to the executeBatch method as a database call. It will show information related to the call in distributed traces.
+  In particular, in the Performance tab for the call it will show the query as "Batch Execute n Queries" where n is the number of queries executed as part of the batch.
+  - For calls to addBatch(String sql), it will represent the number of calls to this method before the executeBatch method is executed. 
+  - For PreparedStatment and CallableStatment it will represent the number of times that addBatch() is called.
+
+summary: |
+  Java agent extension to monitor  the executeBatch method of the Statement, PreparedStatement and CallableStatement interfaces as a database call.
+
+
+icon: icon.svg
```
Comment (jpvajda) [inline quickstarts/java/jdbc-executebatch/config.yml:17]:
```suggestion
```

File: quickstarts/java/jdbc-executebatch/config.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: jdbc-executebatch
+title: JDBC-ExecuteBatch Java Agent Extension
+description: |
+  # What is executeBatch call?
+
+  ExecuteBatch method in jdbc submits a batch of commands to the database for execution and if all commands execute successfully,returns an array of update counts.
```
Comment (jpvajda) [inline quickstarts/java/jdbc-executebatch/config.yml:6]: I'd suggest cleaning up this markdown a bit and using bullets and spacing in the description.

File: quickstarts/java/jdbc-executebatch/config.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+  ExecuteBatch method in jdbc submits a batch of commands to the database for execution and if all commands execute successfully,returns an array of update counts. 
+  The int elements of the array that is returned are ordered to correspond to the commands in the batch, which are ordered according to the order in which they were added to the batch
+    Once deployed the instrumentation will start to track the call to the executeBatch method as a database call. It will show information related to the call in distributed traces.
+  In particular, in the Performance tab for the call it will show the query as "Batch Execute n Queries" where n is the number of queries executed as part of the batch.
+  - For calls to addBatch(String sql), it will represent the number of calls to this method before the executeBatch method is executed. 
+  - For PreparedStatment and CallableStatment it will represent the number of times that addBatch() is called.
+
+summary: |
+  Java agent extension to monitor  the executeBatch method of the Statement, PreparedStatement and CallableStatement interfaces as a database call.
```
Comment (jpvajda) [inline quickstarts/java/jdbc-executebatch/config.yml:14]:
```suggestion
  Java agent extension to monitor the executeBatch method of the Statement, PreparedStatement and CallableStatement interfaces as a database call.
```

File: quickstarts/java/jdbc-executebatch/config.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+authors:
+  - New Relic Labs
+
+documentation:
+  - name: Java agent extension for JDBC ExecuteBatch method
+    description: |
+      Java Agent instrumentation extension for tracking executeBatch method of the Statement, PreparedStatement and CallableStatement interfaces as a database call.
+    url: >-
+      https://github.com/newrelic-experimental/newrelic-java-JDBC-ExecuteBatch
```
Comment (jpvajda) [inline quickstarts/java/jdbc-executebatch/config.yml:38]: This points to an experimental repository, is there actual installation documentation to reference on docs.newrelic.com

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: quickstarts/java/jdbc-executebatch/config.yml (hunk 4 above)
Comment (hsinghkalsi) [reply quickstarts/java/jdbc-executebatch/config.yml:37]: @jpvajda  these assets live in experimental,  we do not have any references in new relic docs.  we use java agent api and  weaver capability to extended java agent instrumentation capabilities.

File: quickstarts/java/jdbc-executebatch/config.yml (hunk 2 above)
Comment (hsinghkalsi) [reply quickstarts/java/jdbc-executebatch/config.yml]: reformatted section to use bullets

**Review by jpvajda — APPROVED** [review]

**Pull request merged**